src/Avalonia.Base/Animation/TransitionObservableBase.cs

```diff
@@ -12,11 +12,11 @@ namespace Avalonia.Animation
     /// <typeparam name="T">Type of the transitioned value.</typeparam>
     public abstract class TransitionObservableBase<T> : SingleSubscriberObservableBase<T>, IObserver<double>
     {
-        private readonly Easing _easing;
+        private readonly IEasing _easing;
```

Comment:
Do we still need `IEasing` or can we just use the `Easing` base class?

Reply:
I'm not sure that we need the base class, it mostly hosts the `Parse` method.

Reply:
@kekekeks we should probably remove the base class, the interface should be enough imo

Reply:
Better to use an abstract base class than an interface because once the interface is part of the public API, it's frozen, whereas we can at least add stuff to the base class.

Reply:
We're going to leave `IEasing` in as it's also used in the renderer. It will be made not client implementable though - custom easings should derive from `Easing`.